test/main.cxx

```diff
@@ -0,0 +1,9 @@
+#include <ifc/version.hxx>
+#include <iostream>
+
+int main(int, char**)
+{
+    std::cout << "Current format version: " << static_cast<unsigned>(ifc::CurrentFormatVersion.major) << '.'
+              << static_cast<unsigned>(ifc::CurrentFormatVersion.minor) << '\n';
+    return 0;
+}
```

Comment:
This is here only to facilitate testing and that the project actually links as documented. The `test` folder is also an entrypoint for CMake, so if you want to be very strict about checking the install interface, then you can do that as well. For now, it's just used from the main project entrypoint via `add_subdirectory`.